std/experimental/allocator/common.d

```diff
@@ -82,10 +84,11 @@ struct Ternary
     }
 }
 
+@safe @nogc nothrow pure
 unittest
 {
     alias f = Ternary.no, t = Ternary.yes, u = Ternary.unknown;
-    auto truthTableAnd =
```

Comment:
I disagree with these changes. `auto` should really be the default everywhere, see http://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/